### PR TITLE
External SPARQL for `taxon`

### PR DESCRIPTION
**Pull request by egonw**
Implements part of https://github.com/fnielsen/scholia/issues/1283

### CODE REVIEW

**Comment by fnielsen** [comment]
@egonw Should the identifier for the parent taxa be parent-taxa rather than info?

**Comment by egonw** [comment]
@fnielsen, good idea! Applied and rebased the patches on the latest master.

**Comment by fnielsen** [comment]
Note to oneself: `git branch -D egonw-feat/external-sparql-taxon`

**Comment by fnielsen** [comment]
@egonw The graph (tree.sparql) has not a parametrized q items. It is homo sapiens.

**Comment by egonw** [comment]
> @egonw The graph (tree.sparql) has not a parametrized q items. It is homo sapiens.

argh... one of my checks, and it still slipped through...

File: scholia/app/templates/taxon_tree.sparql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  ?rgb 
+  ?parent ?parentLabel
+WITH {
+  SELECT ?child ?rgb ?parent WHERE {
+    { 
+      # Parent taxons
+      SELECT ?child ?rgb ?parent  WHERE {
+        SERVICE gas:service {
+          gas:program gas:gasClass "com.bigdata.rdf.graph.analytics.BFS" ;
+                  gas:in wd:Q15978631 ;
```
Comment (fnielsen) [inline scholia/app/templates/taxon_tree.sparql:13]: @egonw I think the home sapiens issue is just here.

Comment (egonw) [reply scholia/app/templates/taxon_tree.sparql]: Actually, in two more lines. See the extra patch.

**Pull request merged**